tests/playwright/assets/api-requests.ts

```diff
@@ -33,6 +33,28 @@ export default class ApiRequests {
 		return id;
 	}
 
+	public async createWithREST( request: APIRequestContext, entity: string, data: PostForREST ) {
```

Comment:
@louiswol94 the original create uses the rest api in a weird way, I mean it calls the index.php file with the rest route. This has never worked with creating meta and I tested it a lot. My approach actually calls REST routes.
I have not changed the original method to avoid breaking something, it looked safer to me to just create a new method, let me know ehat you think

Reply:
@nicoladj77 @louiswol94 
Nice work

Reply:
@nicoladj77 I think it is dependent on your "permalink structure"? Can you share your code for trying to add meta with the `index.php` route?

I imagine it would be something like this:

```
const response = await request.post(`${ this.baseUrl }/index.php`, {
    params: { 
        rest_route: `/wp/v2/${ entity }`
    },
    headers: {
        'X-WP-Nonce': this.nonce,
    },
    data: {
        title: "Your Post Title",
        content: "This is the content of the custom post type.",
        status: "publish",
        meta_input: {
            your_meta_key: "Your meta value"
        }
    }
});
```

Reply:
@louiswol94 it was the exact same code. if you look at the Post type it expects meta as a string. I changed that to object but meta was nto arriving correctly